models/setArticles.models.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../db/connection");
+
+exports.setArticles = () => {
+  const formatedArticles = `
+    SELECT 
+    articles.article_id,
+    articles.title, 
+    articles.author,
+    articles.topic, 
+    articles.created_at,
+    articles.votes,
+    articles.article_img_url,
+    COUNT(comments.comment_id)::int AS comment_count
```

Comment:
`:: int` will do the work, but it'd better if we keep the syntax by using uppercase `:: INT`